src/components/SimpleLink/SimpleLink.js

```diff
@@ -0,0 +1,20 @@
+import React, { PropTypes as toBe } from 'react';
+import './simple-link.scss';
+import Link from 'gatsby-link';
+
+export default class Header extends React.Component {
```

Comment:
![image](https://user-images.githubusercontent.com/1625045/32089298-d6268c04-baf1-11e7-8cdc-ba838aac863d.png)

Reply:
And basically it's not a simple link, it's kinda `target="_blank"`. What do you think about moving `target` to props?

Reply:
good idea